Description: /etc/sysconfig renamed in /etc/default
 On Debian6 the system configuration is in /etc/default, not
 /etc/sysconfig.
Author: Valery Tschopp <valery.tschopp@switch.ch>
Index: emi.argus.pdp/src/main/assembly/fakeroot.xml
===================================================================
--- emi.argus.pdp.orig/src/main/assembly/fakeroot.xml	2012-08-22 10:42:03.000000000 +0200
+++ emi.argus.pdp/src/main/assembly/fakeroot.xml	2012-08-22 10:43:06.000000000 +0200
@@ -83,7 +83,8 @@
         <!-- /etc/sysconfig/argus-pdp optional config -->
         <fileSet>
             <directory>src/main/scripts/sysconfig</directory>
-            <outputDirectory>/etc/sysconfig</outputDirectory>
+            <!-- Debian6 uses /etc/default for system configuration -->
+            <outputDirectory>/etc/default</outputDirectory>
             <fileMode>0644</fileMode>
         </fileSet>
         <!-- /etc/rc.d/init.d startup script -->
